### PR TITLE
CMR-5166: As a Client User, I want to retrieve granules in UMM-G format.

### PR DESCRIPTION
**Pull request by ygliuvt**

### CODE REVIEW

File: system-int-test/test/cmr/system_int_test/search/granule_retrieval_test.clj [hunk 1]
```diff
@@ -35,17 +45,85 @@
...
+          (is (= (:metadata umm-g-gran) (:body response))))
+
+        "without specifying accept format"
+        nil
+
+        "specifying umm json accept format without version"
+        "application/vnd.nasa.cmr.umm+json"
+
+        "specifying umm json accept format with version"
+        "application/vnd.nasa.cmr.umm+json;version=1.4"))
```
Comment (chris-durbin) [inline system-int-test/test/cmr/system_int_test/search/granule_retrieval_test.clj]: Do we already have tests that verify all potential versions can be returned successfully? It seems like that would be worthwhile.

**Comment by chris-durbin** [comment]
> I think cmr.umm-spec.legacy is probably not a good namespace name because legacy means other things for CMR.
As a matter of personal principle I refuse to work on any code with 'legacy' in the name.

File: system-int-test/test/cmr/system_int_test/search/granule_retrieval_test.clj (hunk 1 above)
Comment (ygliuvt) [reply system-int-test/test/cmr/system_int_test/search/granule_retrieval_test.clj]: There is only one version for UMM-G and it is `1.4` oddly.

**Comment by ygliuvt** [comment]
>> I think cmr.umm-spec.legacy is probably not a good namespace name because legacy means other things for CMR.

> As a matter of personal principle I refuse to work on any code with 'legacy' in the name.

The only way I can work on legacy code and not lose my mind is to not bother with changing their names. :) I really have no idea why this namespace was named `legacy`.

**Pull request merged**